modules/layers/src/layers/editable-h3-layer.ts

```diff
@@ -0,0 +1,152 @@
+/* eslint-env browser */
+
+import { H3ClusterLayer } from '@deck.gl/geo-layers';
+import { ViewMode } from '@nebula.gl/edit-modes';
+import { polyfill } from 'h3-js';
+import EditableGeoJsonLayer from './editable-geojson-layer';
+import EditableLayer from './editable-layer';
+
+const DEFAULT_EDIT_MODE = ViewMode;
+const DEFAULT_H3_RESOLUTION = 9;
+
+const defaultProps = {
+  mode: DEFAULT_EDIT_MODE,
+
+  // EditableGeoJsonLayer
+  geoJsonData: {},
+  ...EditableGeoJsonLayer.defaultProps,
+
+  // h3 layer
+  h3Data: {},
+  filled: false,
+  stroked: true,
+  lineWidthScale: 1,
+  lineWidthMinPixels: 1,
+  lineWidthMaxPixels: Number.MAX_SAFE_INTEGER,
+  lineWidthUnits: 'pixels',
+  getHexagons: (d) => d.hexIds,
+  resolution: DEFAULT_H3_RESOLUTION,
+};
+
+export default class EditableH3Layer extends EditableLayer {
```

Comment:
I think we can make this closer to existing APIs by doing the following:
1. rename to `EditableH3ClusterLayer`.
2. take a `selectedIndexes` prop which is an array of indexes into the `data` array. It's ok if you only support single selection today (`selectedIndexes.length === 1`), but taking an array of indexes future-proofs the API a bit. And it will support operations in the future that may operate on multiple selection (e.g. translate, scale, etc).

```
new EditableH3ClusterLayer({
  data: [{h3Cells: [...]}], // valid
  data: [{h3Cells: [...]}, {h3Cells: [...]}], // valid? (based on how much time you have)
  selectedIndexes: [0], // valid
  selectedIndexes: [1], // valid? (based on how much time you have)
  selectedIndexes: [0, 1], // invalid
  selectedIndexes: [], // invalid? (based on how much time you have)
  booleanOperation: null, // valid? (based on how much time you have)
  booleanOperation: 'union', // valid
  ...
})
```